src/schema/rules/datatypes/micr.yaml

```diff
@@ -24,6 +24,7 @@ microscopy:
   - .png
   - .tif
   - .json
+  - .ngff
```

Comment:
I would place `.ngff` next to the OME-TIFF files to indicate priority. Also, should this have a trailing slash? I didn't check the other directory formats in datatypes...

```suggestion
  - .ngff/
  - .png
  - .tif
  - .json
```

Reply:
We do have trailing slashes for the directory formats.